release_notes.md

```diff
@@ -8,24 +8,8 @@
   - Fix thread context classloader for middleware chain
 - Update Java Worker Version to [2.15.0](https://github.com/Azure/azure-functions-java-worker/releases/tag/2.15.0):
   - Update grpc-protobuf to 1.64.0 and application insights agent version to 3.5.2
-- Marking SyncTriggers [RequiresRunningHost]. (#10233)
```

Comment:
Can you clarify why some of the release notes are being removed? Was this not reset during the last release?

Reply:
This commit mentioned in the release notes for this PR was already included in version 4.35.x, which is why it has been excluded from the release notes for the upcoming version 4.36.x.

Reply:
Can you clarify where you're pulling the release notes from? I wouldn't expect this state as part of release prep.

Reply:
@fabiocav  Typically, we don't include the addition process, but this time we've meticulously cross-checked the list of commits made after the 4.35.0 release. We then compared it with the release notes of 4.35.4 and filtered it from the existing release notes.
https://github.com/Azure/azure-functions-host/releases
![image](https://github.com/user-attachments/assets/6b80b0a5-62cd-4474-a2b8-f883c0125279)

Reply:
Updated the release notes.